### PR TITLE
Added check for incorrect ifConnectorPresent truth values, if invalid…

### PR DESCRIPTION
**Pull request by craig-nokia**
Some SNMP devices (running older software) do not initialized the ifConnectorPresent correctly.  

IF-MIB Description: This object has the value 'true(1)' if the interface sublayer has a physical connector and the value 'false(2)' otherwise.  

This patch only allows true and false.  If any other value, force it to false.

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/polling/ports.inc.php [hunk 1]
```diff
@@ -620,6 +620,11 @@
             $this_port['ifVlan'] = $this_port['dot1qPvid'];
         }
 
+        // Set ifConnectorPresent to null when the device does not support IF-MIB truth values.
+        if (isset($this_port['ifConnectorPresent']) && !in_array($this_port['ifConnectorPresent'], array('true', 'false'))) {
+            $this_port['ifConnectorPresent'] = 'null';
```
Comment (murrant) [inline includes/polling/ports.inc.php:625]: This should be null without quotes.

Comment (murrant) [reply includes/polling/ports.inc.php:625]: also, please use short array syntax ['true', 'false']

Comment (craig-nokia) [reply includes/polling/ports.inc.php]: Done.

**Comment by Jellyfrog** [comment]
Closing due to inactivity, feel free to reopen

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-66-release-changelog-july-2020/12953/1

**Pull request merged**